### ISSUE TITLE
"Minimum" parameter is being missed when v.min = 0

### ISSUE THREAD

**Issue by pyloolex** [maximkulkin/lollipop-jsonschema#9]
I have the following piece of the code:
```python
import lollipop.types as s
import lollipop.validators as v
import lollipop_jsonschema

schema = s.Object({
    'param': s.Optional(s.Integer(validate=v.Range(min=0)))
})

print lollipop_jsonschema.Encoder().json_schema(schema)
```

Expected output: 
```
OrderedDict([('type', 'object'), ('properties', OrderedDict([('param', OrderedDict([('type', 'integer'), ('minimum', 0), ('default', None)]))]))])
```

Actual: 
```
OrderedDict([('type', 'object'), ('properties', OrderedDict([('param', OrderedDict([('type', 'integer'), ('default', None)]))]))])
```

I guess, the problem is in this line - [lollipop_jsonschema/jsonschema.py:172
](https://github.com/maximkulkin/lollipop-jsonschema/blob/master/lollipop_jsonschema/jsonschema.py#L172)

Would it make sense to replace `any(v.min for ...` with `any(v.min is not None for ...`?

**Comment by akscram** [comment]
It seems that your suggestion according to `any(...)` is correct, however I would suggest to filter `None` values first and then only if the remaining list is not empty pass them into `min()` or `max()` respectively, e.g.:
```
min_values = filter(lambda v: v.min is not None, range_validators)
if min_values:
    js['minimum'] = max(min_values)
```
The same thing for `maximum`.

**Comment by maximkulkin** [comment]
Fixed in 0.8.2
